tests/chainer_tests/functions_tests/math_tests/test_average.py

```diff
@@ -47,7 +47,8 @@ def setUp(self):
         # Sample weights. Weights should not sum to 0.
         while True:
             self.w = numpy.random.uniform(-2, 2, w_shape).astype(self.dtype)
-            if self.w.sum() > 1e-2:
+            w_sum_eps = 0.2 if self.dtype == numpy.float16 else 5e-2
+            if self.w.sum() > w_sum_eps:
```

Comment:
Test `abs(self.w.sum())` to fix the slip in #3704.

Reply:
Fixed.